docs/katana.property_graph.rst

```diff
@@ -1,5 +1,6 @@
-Property Graphs (katana.property_graph)
-=======================================
+===============
+Property Graphs
+===============
```

Comment:
If we are standardizing on "above and below" heading style, can we have a checker for it at some point? (Also I personally slightly prefer the under only style, but I don't really care much.)

Reply:
I actually don't care about the style either but from my reading of

- https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

over and underlining is a different section size than just underling, so I thought to use that for titles of pages so that the rest of the page has more section styles to use.

Reply:
ooh. Thanks for mentioning this. I wasn't careful enough. I agree with your change.